content/guides/java/intro.md

```diff
@@ -37,14 +37,13 @@ Replace `opentracing-api` with the `opentracing-noop`, `opentracing-mock` or `op
 
 #### Main API
 
-The main OpenTracing API declares all the main components as interfaces, which additional helper classes.
+The main [OpenTracing API](http://javadoc.io/doc/io.opentracing/opentracing-api/0.31.0) declares all the main components as interfaces, which additional helper classes, such as **Tracer**, **Span**, **SpanContext**, **Scope**, **ScopeManager** and **Format** (used to define common **SpanContext** extraction/injection formats).
```

Comment:
If you link to `http://javadoc.io/doc/io.opentracing/opentracing-api`, it will redirect automatically to the latest.